types/readable-stream/index.d.ts

```diff
@@ -7,8 +7,6 @@
 
```

Comment:
Do we know how to bump the version of `@types/readable-stream`? Is it this:

```
// Type definitions for readable-stream 2.3.1
```

Reply:
I think that bumping is automatic through typescript-bot when it is going to release this PR. Major and minor didn't change, so I guess it will publish a patch.